dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/AppsResourceTest.java

```diff
@@ -477,39 +449,36 @@ public void Test_Create_App_Descriptor_Then_Create_App_Integration_Then_Delete_A
                         .getAppDetail(request, response, appKey, siteId);
                 Assert.assertEquals(HttpStatus.SC_NOT_FOUND, responseNotFound.getStatus());
             }
-        } catch (Exception e) {
-            Logger.error(AppsResourceTest.class, e);
-            fail();
         }
     }
 
     /**
      * Test hidden secrets come back protected.
      * Given scenario: We have a descriptor with hidden and non-hidden stuff
-     * Expected Result: Whatever was marked as hidden in the app-descriptor must come back protected.
+     * Expected Result: Whatever was marked as hidden in the app-descriptor must come back protected Also we expect the response to have the params in natural order.
      */
     @Test
-    public void Test_Protected_Hidden_Secret_And_Values_Returned_Match_Descriptor() {
+    public void Test_Secret_Serializer_Returned_Values_Match_Descriptor_Verify_Hidden_Secrets_Are_Protected_Verify_Params_Are_Ordered() throws Exception{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5358201)